test/WakuRlnV2.t.sol

```diff
@@ -1,15 +1,18 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.19 <0.9.0;
 
-import { Test, console } from "forge-std/Test.sol";
+import { Test } from "forge-std/Test.sol";
+import { stdStorage, StdStorage } from "forge-std/Test.sol";
 
 import { Deploy } from "../script/Deploy.s.sol";
 import { DeploymentConfig } from "../script/DeploymentConfig.s.sol";
-import { WakuRlnV2 } from "../src/WakuRlnV2.sol";
+import "../src/WakuRlnV2.sol";
```

Comment:
Just mentioning this: Typically it's easier to follow where stuff comes  from with named imports. No biggie at all here, but if there isn't a good reason to make this change I'd rather keep it as it was.

Reply:
Yeah there's a bunch of errors that we assert for, thats why grouped all the imports together :)